lib/features/profile/view/profile_view.dart

```diff
@@ -39,115 +43,17 @@ class ProfileBody extends StatelessWidget {
-                                .textTheme
-                                .bodyMedium
-                                ?.copyWith(
-                                    fontWeight: FontStyleGuide.fwBold,
-                                    fontSize: FontStyleGuide.fontSize16));
-                      }),
-                      const SizedBox(height: HsDimens.spacing4),
-                      Text(HatSpaceStrings.current.viewProfile,
-                          style: Theme.of(context).textTheme.bodyMedium),
-                      const SizedBox(height: HsDimens.spacing6),
-                      BlocSelector<GetUserDetailCubit, GetUserDetailState,
-                          List<Roles>>(selector: (state) {
-                        if (state is GetUserDetailSucceedState) {
-                          return state.roles;
-                        }
-
-                        return [];
-                      }, builder: (_, roles) {
-                        return Wrap(
-                          spacing: HsDimens.spacing4,
-                          runSpacing: HsDimens.spacing4,
-                          children: roles
-                              .map((role) => Container(
-                                    padding: const EdgeInsets.symmetric(
-                                      horizontal: HsDimens.spacing8,
-                                      vertical: HsDimens.spacing4,
-                                    ),
-                                    decoration: ShapeDecoration(
-                                      shape: const StadiumBorder(),
-                                      color: (() {
-                                        switch (role) {
-                                          case Roles.tenant:
-                                            return HSColor.blue05;
-                                          case Roles.homeowner:
-                                            return HSColor.orange05;
-                                        }
-                                      }()),
-                                    ),
-                                    child: Text(role.title,
-                                        style: Theme.of(context)
-                                            .textTheme
-                                            .bodySmall
-                                            ?.copyWith(
-                                                color: HSColor.neutral1)),
-                                  ))
-                              .toList(),
-                        );
-                      }),
-                    ],
-                  ),
-                ),
-                const SizedBox(width: HsDimens.spacing12),
-                SvgPicture.asset(Assets.icons.chevronRight)
-              ],
-            ),
-          ),
-        ),
+        BlocConsumer<ProfileCubit, ProfileState>(listener: (_, state) {
+          if (state is DeleteAccountSucceedState) {
+            Navigator.of(context)
```

Comment:
1. Use router.dart 
2. This will place sign up screen on top of Dashboard while showing profile page which blank UI. What will happen when user taps on Skip button in sign up screen?